sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -276,13 +279,23 @@ private void createPrePopulatedDocs(int numberOfPreCreatedDocuments) {
                     PojoizedJson x =
                         resp.getItem();
                     return x;
+                }).onErrorResume(throwable -> {
+                    failureCount.incrementAndGet();
+                    logger.error("Error during pre populating item "+throwable.getMessage());
```

Comment:
please don't do concat in the logger
```suggestion
                    logger.error("Error during pre populating item", throwable.);
```

Reply:
Done